### DISCUSSION TITLE
Enlever missions

### DISCUSSION THREAD

**Discussion by OrionYp** [Crownicles/Crownicles#2531] [Nouvelles idées]
Pouvoir retirer une mission secondaire gratuitement par jour dans le /magasinmission 
Parce que c'est trop chiant d'avoir des missions gagnez des points de rage/combats de boss juste après avoir fait l'île..
![image](https://github.com/user-attachments/assets/f3580ef4-0d96-4edf-963c-8dc4a6b8c13f)
J'ai fait l'île hier soir, 2h après j'ai 2 nouvelles missions pour l'île donc je suis obligé de payer des gemmes (5 pour changer une mission ou 15 pour un nouveau voyage qui ne me garantit pas de les réussir) ce que je ne veux pas faire donc je ne peux pas avoir de nouvelles missions secondaires pendant plusieurs jours.

**Comment by OrionYp** [comment]
Si vous voulez proposer autre chose qui permet de faire disparaître ces missions sans payer de gemmes, ni attendre plusieurs jours, n'hésitez pas

**Comment by BastLast** [comment]
c'est littéralement le principe du passe mission

**Reply by OrionYp** [reply]
Le passe-mission a été remplacé par le change-mission et est giga cher (10 💎).

**Reply by BastLast** [reply]
yep c'est fait exprès

**Reply by BastLast** [reply]
sinon les gens feraient jamais les missions longues
